apps/spectator/app/game/[id]/_components/CheerTalk/_components/MenuModal/MenuModal.css.ts

```diff
@@ -1,17 +1,20 @@
 import { rem, theme } from '@hcc/styles';
 import { style } from '@vanilla-extract/css';
+import { calc } from '@vanilla-extract/css-utils';
 
 export const container = style({
   display: 'flex',
   flexDirection: 'column',
   width: '100%',
-  maxWidth: rem(218),
-  marginInline: 'auto',
-  paddingInline: theme.spaces.default,
+  maxWidth: calc.divide(theme.sizes.appWidth, 2),
+  margin: '0 auto',
+  padding: `0 ${rem(16)}`,
```

Comment:
```suggestion
  marginInline: 'auto',
  paddingInline: theme.spaces.default,
```

불필요하게 템플릿 리터럴을 사용하는 것을 줄일 수 있을 것 같아요!